### PR TITLE
Add source code documentation of justified navbar for #15906

### PR DESCRIPTION
**Pull request by kkirsche**
Document in the source code that the justified nav is meant for single line list items only. This should help prevent Issues like #15906 in the future.

cc: @cvrebert / @mdo

### CODE REVIEW

**Comment by cvrebert** [comment]
http://getbootstrap.com/components/#nav-justified would be the more important place to state this, imho.

**Comment by cvrebert** [comment]
And ignore that now that I read your other comment.

**Comment by kkirsche** [comment]
:+1: no worries. This is certainly an odd situation that has been hard to follow and discuss due to how similar it is to other components. :smiley:
